launch/test/launch/test_execute_process.py

```diff
@@ -84,3 +86,35 @@ def on_exit_function(context):
     assert 0 == ls.run()
     assert on_exit_callback.called
     assert on_exit_function.called
+
+
+def test_execute_process_with_respawn():
+    """Test launching a process with a respawn and respawn_delay attribute."""
+    def on_exit_callback(event, context):
+        on_exit_callback.called_count = on_exit_callback.called_count + 1
+    on_exit_callback.called_count = 0
+
+    respawn_delay = 2
```

Comment:
@iuhilnehc-ynos nit^N: 
```suggestion
    respawn_delay = 2.0
```

Reply:
Done